compiler/src/iree/compiler/Dialect/Stream/Transforms/PropagateTimepoints.cpp

```diff
@@ -242,20 +250,18 @@ static void expandRegion(Region &region, bool canModifyEntryBlock,
       if (!isResourceType(resourceArg.getType()))
         continue;
       auto timepointArg =
-          block.insertArgument(i, timepointType, resourceArg.getLoc());
+          block.insertArgument(i + 1, timepointType, resourceArg.getLoc());
```

Comment:
`i + 1`? Was there an off by one error here before, or is this now accounting for some change in the IR?

Reply:
transposed it from (timepoint, resource) to (resource, timepoint) so that the tied operand remains in the same relative order